### PR TITLE
Update to draft 3

### PR DESCRIPTION
**Pull request by sdelatorrep**

### CODE REVIEW

File: default_cohort_schema.yaml [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+externalDocs:
+  description: |
+    This object contains info from a set of persons who satisfy one or more
+    inclusion criteria for a duration of time.
+  url: 'http://beacon-project.io/'
+paths: {}
+components:
+  schemas:
+    CollectionEvent:
```
Comment (silverdaz) [inline default_cohort_schema.yaml:21]: Camel-case on the first letter too ?

File: default_cohort_schema.yaml [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+              description: data availability
+              type: boolean
+            availabilityCount:
+              description: count of individuals with data available
+              type: integer
+            distribution:
+              description: TBD # TBD
+              type: array
+              # TBD
+    Cohort:
```
Comment (silverdaz) [inline default_cohort_schema.yaml:153]: same here?

File: default_variant_annotations_schema.yaml [hunk 3]
```diff
@@ -17,9 +17,9 @@ externalDocs:
 paths: {}
 components:
   schemas:
-    genomicFeatures:
+    GenomicFeatures:
```
Comment (silverdaz) [inline default_variant_annotations_schema.yaml:20]: Are you really sure about those changes? I can see an inconsistency between JSON levels.

**Review by silverdaz — APPROVED** [review]
Strange camel-casing in names that I find inconsistent from level to level.
Still approving the updates.

File: default_cohort_schema.yaml (hunk 1 above)
Comment (MrRobb) [reply default_cohort_schema.yaml:21]: UpperCamelCase for schemas, camelCase for properties

File: default_cohort_schema.yaml (hunk 2 above)
Comment (MrRobb) [reply default_cohort_schema.yaml:153]: UpperCamelCase for schemas, camelCase for properties

**Comment by MrRobb** [comment]
I used UpperCamelCase for schemas and camelCase for properties. I found it to be the most consistent pattern.

**Pull request merged**